src/dynamicTypes/eventHandlersService.ts

```diff
@@ -0,0 +1,21 @@
+import { EventHandler, ComponentsEventHandlers } from "../../scripts/types";
+import Constants from "../../scripts/constants";
+import eventHandlersJSON from "../../dist/eventHandlers.json";
```

Comment:
what about fetching this dynamically?

Reply:
This will require a lot more work. I might do it in a later PR